shopinvader/services/sale.py

```diff
@@ -49,6 +51,16 @@ def ask_email_invoice(self, _id):
         self._ask_email_invoice = True
         return self.ask_email(_id)
 
+    def cancel(self, _id):
+        order = self._get(_id)
+        self._cancel(order)
+        return self._to_json(order)[0]
+
+    def redraft(self, _id):
+        order = self._get(_id)
+        self._cancel(order, redraft=True)
+        return self._to_json(order)[0]
```

Comment:
You should also update the session in order to set the id of this sale as the cart id

Reply:
Thanks, done (also I renamed `redraft` to `reset_to_cart`)